src/components/custom-field-input-multiple-choice/custom-field-input-multiple-choice.jsx

```diff
@@ -0,0 +1,57 @@
+import React, {
```

Comment:
Do we want to inline this or is the line too long?

Reply:
I'm prefer multi-lining things to be a bit more git friendly! if we add or remove named imports then we don't mess with the other lines.